e2e/testsuite/testsuite.go

```diff
@@ -195,6 +197,25 @@ func (s *E2ETestSuite) BroadcastMessages(ctx context.Context, chain *cosmos.Cosm
 	return resp, err
 }
 
+// RegisterCounterPartyPayee broadcasts a MsgRegisterCounterpartyPayee message.
+func (s *E2ETestSuite) RegisterCounterPartyPayee(ctx context.Context, chain *cosmos.CosmosChain,
+	user *ibc.Wallet, portID, channelID, relayerAddr, counterpartyPayeeAddr string) (sdk.TxResponse, error) {
+	msg := feetypes.NewMsgRegisterCounterpartyPayee(portID, channelID, relayerAddr, counterpartyPayeeAddr)
+	return s.BroadcastMessages(ctx, chain, user, msg)
+}
+
+// PayPacketFeeAsync broadcasts a MsgPayPacketFeeAsync message.
+func (s *E2ETestSuite) PayPacketFeeAsync(
+	ctx context.Context,
+	chain *cosmos.CosmosChain,
+	user *ibc.Wallet,
+	packetID channeltypes.PacketId,
+	packetFee feetypes.PacketFee,
+) (sdk.TxResponse, error) {
+	msg := feetypes.NewMsgPayPacketFeeAsync(packetID, packetFee)
+	return s.BroadcastMessages(ctx, chain, user, msg)
+}
```

Comment:
Should these go into a ICS29 test suite later? Even if the test suite doesn't have any of its own tests, but just provides access to these helpers?

Reply:
that could be a good idea, the only dependency is `BroadcastMessages` from E2ETestSuite. Since I'm moving this stuff around now let me take a stab and extracting these out in this PR.

Reply:
@colin-axner after taking a look, I think this will require more additional shuffling than I would like to do in this PR. I'll create a separate issue for this as it should definitely still be done!